setup.py

```diff
@@ -49,6 +49,12 @@ def _export_version(version, sha):
 
 print('-- Building version ' + VERSION)
 
+pytorch_package_version = os.getenv('PYTORCH_VERSION')
+
+pytorch_package_dep = 'torch'
+if pytorch_package_version is not None:
+    pytorch_package_dep += "==" + pytorch_package_version
```

Comment:
Does the cuda version etc factor in here?

Reply:
No it does not, but then again we don't account for cuda version in either `audio` or `vision` either